pygdf/_gdf.py

```diff
@@ -136,6 +136,7 @@ def apply_sort(col_keys, col_vals, ascending=True):
     'left': libgdf.gdf_left_join_generic,
     'inner': libgdf.gdf_inner_join_generic,
     'outer': libgdf.gdf_outer_join_generic,
+    'multi-left': libgdf.gdf_multi_left_join_generic,
```

Comment:
I think we should make 'left' default to the `libgdf.gdf_multi_left_join_generic` and have a 'left-compat' that maps to `libgdf.gdf_left_join_generic`.